README.md

```diff
@@ -150,20 +150,49 @@ We collaborate closely with the design and product team. The design team provide
 
 ## Deployment
 
+### Git Branching Model
+
+- When a Github issue is started, create a branch with the `feature/` prefix. Every push to branches with this prefix will automatically deploy to the review AWS environment. See the [Review branches](#review-branches) section for details on how to access this build.
+
+- After a pull request for a `feature/` branch is approved, it is merged into the `staging` branch. This will trigger an automatic deployment to the staging AWS environment. Note that changes may take several minutes to take effect. The staging app is accessible at http://staging.fightpandemics.work.
+
+- When we are ready to release to production, a pull request is opened to merge the `staging` branch to `production`. After this pull request is approved and merged, it will trigger an automatic deployment to the production AWS environment. Note that changes may take several minutes to take effect. The production app is accessible at http://fightpandemics.com
+
+- A `hotfix/` branch is created if there is a critical fix that needs to be deployed to production as soon as possible, but staging has untested code that is not safe to deploy to production.
+    - This branch must be created off the `production` branch.
+    - Every push to `hotfix/` branches will automatically deploy to the review AWS environment. See the [Review branches](#review-branches) section for details on how to access this build.
+    - Two pull requests must be opened for hotfixes: one PR to merge into `production` and one PR to merge into `staging`.
+    - Hotfixes must be tested in the staging AWS environment before being merged to the `production` and `staging` branches. In order to deploy only the hotfix to the staging enviroment, create and push a git tag with a `hotfix-` prefix. This will trigger a deployment to the staging environment: `git tag -a hotfix-<GITHUB_ISSUE_NUMBER> && git push origin hotfix-<GITHUB_ISSUE_NUMBER>`.
```

Comment:
I am wondering about it here like if we merge the hotfix to prod and to staging and then we will merge staging to prod, we will have two commits that are doing the same thing in prod right? Should be fine but just wondering if I am thinking correctly. So we won't be back merging prod into staging, but instead, merge hotfixes to both staging and prod...

Reply:
Actually that's a good point; if we use the Github `Squash and merge` button to merge the hotfix branch into both staging and production, we'll have two distinct commit hashes for the same change. It's probably better to backmerge from production to staging instead to get the hotfix into staging. I'll update this.